src/conic_algorithm.jl

```diff
@@ -77,37 +73,42 @@ type PajaritoConicModel <: MathProgBase.AbstractConicModel
-    A_cont::SparseMatrixCSC{Float64,Int64} # Submatrix of A containing full rows and continuous variable columns
-    b_full::Vector{Float64}     # Subvector of b containing full rows
+    var_types::Vector{Symbol}   # Variable types vector on original variables (only :Bin, :Cont, :Int)
+    var_start::Vector{Float64}  # Variable warm start vector on original variables
+
+    # Transformed data
+    cone_con_new::Vector{Tuple{Symbol,Vector{Int}}} # Constraint cones data after converting variable cones to constraint cones
+    cone_var_new::Vector{Tuple{Symbol,Vector{Int}}} # Variable cones data after converting variable cones to constraint cones and adding slacks
+    num_con_new::Int            # Number of constraints after converting variable cones to constraint cones
+    num_var_new::Int            # Number of variables after adding slacks
+    b_new::Vector{Float64}      # Subvector of b containing full rows
+    c_new::Vector{Float64}      # Objective coefficient subvector for continuous variables
+    A_new::SparseMatrixCSC{Float64,Int64} # Submatrix of A containing full rows and continuous variable columns
+    row_to_slckj::Dict{Int,Int} # Dictionary from row index in MIP to slack column index if slack exists
+    row_to_slckv::Dict{Int,Float64} # Dictionary from row index in MIP to slack coefficient if slack exists
+    num_cone_nlnr::Int          # Number of nonlinear cones
 
     # MIP constructed data
     model_mip::JuMP.Model       # JuMP MIP (outer approximation) model
-    x_bint::Vector{JuMP.Variable} # JuMP integer variables vector
-    x_all::Vector{JuMP.Variable} # JuMP vector of all variables for heuristics and warm-starting
-    num_cone_nlnr::Int          # Count of nonlinear cones
-    map_ifvar::Vector{Bool}     # Bool for whether nonlinear cone is a variable cone in the conic problems
+    x_mip::Vector{JuMP.Variable} # JuMP vector of original variables
     map_spec::Vector{Symbol}    # Species of nonlinear cone
-    map_ind::Vector{Vector{Int}} # Indices (rows or columns) of cone in original (conic relaxation) data
-    map_ind_new::Vector{Vector{Int}} # Indices (rows or columns) of cone in transformed (conic subproblem) data
-    map_Asub::Vector{SparseMatrixCSC{Float64,Int64}} # Submatrix of A_cont containing rows corresponding to nonlinear constraint cone
-    map_vars::Vector{Vector{JuMP.Variable}} # Variables associated with nonlinear cone
-    map_vars_dagg::Vector{Vector{JuMP.Variable}} # Disaggregated variables associated with nonlinear cone
-    map_vars_help::Vector{Vector{JuMP.Variable}} # Helper variables associated with nonlinear cone
-    map_vars_smat::Vector{Array{JuMP.Variable,2}} # SDP original and helper variables in smat form
+    map_rows::Vector{Vector{Int}} # Row indices in MIP for nonlinear cone
+    map_dim::Vector{Int}        # Dimension of nonlinear cone
+    map_vars::Vector{Vector{Vector{JuMP.Variable}}} # JuMP variables associated with slacks of nonlinear cone
```

Comment:
three levels of nesting is confusing. could you give more info in the comment, e.g.
```
map_vars[i][j][k] is the kth variable of the jth slack of the ith nonlinear cone
```
or whatever